Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs

```diff
@@ -460,6 +460,32 @@ public void Wzhzhzh(EntityUid uid, MicrowaveComponent component, EntityUid? user
             UpdateUserInterfaceState(uid, component);
         }
 
+        public void StopCooking(Entity<MicrowaveComponent> ent, bool failure)
+        {
+            if (!TryComp<ActiveMicrowaveComponent>(ent, out var activeMicrowave))
+                return;
+
+            RemComp<ActiveMicrowaveComponent>(ent);
```

Comment:
Move this to later just in case it gets deleted before the other code using its fields get run.

Reply:
Done.